### ISSUE TITLE
Add support for `behavior`

### ISSUE THREAD

**Issue by t11r** [tify-iiif-viewer/tify#190]
See https://iiif.io/api/presentation/3.0/#behavior

This should be overrideable via config.

**Comment by cmahnke** [comment]
See #181

**Comment by cmahnke** [comment]
Make sure 
```
"behavior": [
    "individuals"
  ],
```
hides Button for double sided view.

**Comment by t11r** [comment]
Will be fully supported in the next release.

**Comment by t11r** [comment]
Supported since [TIFY 0.34](https://github.com/tify-iiif-viewer/tify/releases/tag/v0.34.0).

**Comment by frederik-elwert** [comment]
`"behavior": ["continuous"]` is still not supported properly. Compare the rendering of the cookbook recipe:

* In TIFY: <https://tify.rocks/?manifest=https%3A%2F%2Ftify.rocks%2Fmanifests%2Fiiif-cookbook-collection.json&tify=%7B%22childManifestUrl%22%3A%22https%3A%2F%2Fiiif.io%2Fapi%2Fcookbook%2Frecipe%2F0011-book-3-behavior%2Fmanifest-continuous.json%22%2C%22view%22%3A%22export%22%7D>
* In Mirador: <https://projectmirador.org/embed/?iiif-content=https://iiif.io/api/cookbook/recipe/0011-book-3-behavior/manifest-continuous.json>

Mirador correctly displays the manuscript as a single continuous image, while Tify still uses a paged display mode.

Should this issue be re-opened (I don’t have the rights for that), or should I create a new issue?

(Background: For a collection website, I want to use Tify, but I currently have to fall back to mirador for manifests with continuous display, so I have a concrete interest in seeing this supported.)

**Comment by t11r** [comment]
Indeed, `continuous` is not yet supported, as well as some other behaviors, and I cannot give you a date when it will be yet.

Could you add an URL so we have a real-world use case for testing?

**Comment by frederik-elwert** [comment]
Thanks for the reply!

We are using the feature to reconstruct image sequences from collections. Our actual use case is still under development, but a demo is already online:

https://frederik-elwert.github.io/pelican-collection-demo/exhibit-c-dynamic-image-sequences.html

The Manifest URL: https://frederik-elwert.github.io/pelican-collection-demo/images/derivatives/iiif/sequence1/manifest.json

While the site generally uses Tify as its IIIF viewer, for this functionality we fall back to Mirador.

**Comment by t11r** [comment]
Thanks! We decided to prioritize this feature.
